Campaign/management/commands/StartNewCampaign.py

```diff
@@ -8,6 +8,7 @@
 
 from django.core.management.base import BaseCommand, CommandError
 from django.core.files import File
+from django.core.files.base import ContentFile

```

Comment:
Weird, why GitHub is showing this change if it has been already made and is in the master branch?